src/components/tables/TableConnected.tsx

```diff
@@ -57,16 +59,46 @@ export const getTableCompositeState = (state: IReactVaporState, id: string): ITa
   } as ITableCompositeState;
 };
 
+const getDataById = (data: ITableData, props: ITableOwnProps): ITableById => data.byId;
+
+const getSelectedIds = (data: ITableData, props: ITableOwnProps): string[] => data.selectedIds || [];
+
+const getActions = (data: ITableData, props: ITableOwnProps): (rowData?: IData) => IActionOptions[] => props.getActions;
+
+const getMultiSelect = (data: ITableData, props: ITableOwnProps): boolean => props.rowsMultiSelect;
+
+const actionsSelector = createSelector(
+  [getDataById, getSelectedIds, getMultiSelect, getActions],
+  (byId: ITableById, selectedIds: string[], isMultiSelect: boolean, getAction: (rowData?: IData) => IActionOptions[]): IActionOptions[] => {
```

Comment:
getAction --> getActions

Reply:
I get a lint error because getActions is defined 2 time. weird

Reply:
oh I see,surely because of your `getActions` above, since its the ownProps getActions, you could name this `ownPropsGetActions` not to confuse this one with the one declared above